distribution/changelog.txt

```diff
@@ -100,6 +100,7 @@
 - Fix: [#6846] Zoom level in some ride overview windows was erroneously set too high.
 - Fix: [#6904] Manually added multiplayer servers not saved.
 - Fix: [#7003] Building sloped paths through flat paths with clearance checks off causes glitches.
+- Fix: [#7011]: Swinging and bobsleigh cars going backwards swing in the wrong direction (original bug).
```

Comment:
Oops, can you drop the second colon?

Reply:
My bad, should have noticed that 👎 Fixed.